src/trodes_to_nwb/convert_ephys.py

```diff
@@ -87,6 +87,8 @@ def __init__(
         else:
             self.nwb_hw_channel_order = nwb_hw_channel_order
 
+        if stream_index == 3 and len(self.nwb_hw_channel_order) < self.n_channel:
```

Comment:
Why do you need to check for the stream index?

Reply:
You use different stream indices for getting data from ephys, digital input, analog headstage, etc.  3 is the one that corresponds to the ephys data, so is the case where you need to check this

Reply:
How well documented is this? Maybe a comment here would help?

Reply:
It could use some better explanation in the code.  I'll comment here but also add some other changes to make the use of the stream index more interpretable overall

Reply:
Great. I think any way there can be documentation of the code so that future maintainers will understand will be a big help.

Reply:
Made the stream selection more intuitive by allowing use of stream name instead of only index.